pkg/apiserver/middleware_logger.go

```diff
@@ -59,6 +61,11 @@ func LoggingMiddleware(logger mon.Logger) gin.HandlerFunc {
 		}
 
 		for _, e := range ginCtx.Errors {
+			var notFound db_repo.RecordNotFoundError
```

Comment:
the logger middleware shouldn't be aware of these special cases..  
and there is no need for it, if we handle it in the crud handler the right way..